### PR TITLE
Adding plot

### PR DESCRIPTION
**Pull request by SethBailey**
adding a back ground colour option

### CODE REVIEW

File: Config/Medicine.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 plaster, 30, 10, Perfect for all flesh wounds
 tea, 50, 25, the real elixer of life
-bandade, 70, 50, Should stop the bleeding from most cuts
+bandaid, 70, 50, Should stop the bleeding from most cuts
```
Comment (Stackingit) [inline Config/Medicine.csv:3]: Looks like bandaid should be two words ... band aid

File: Program.cs [hunk 2]
```diff
@@ -64,6 +64,31 @@ public void Begin()
             TypeWriter.WriteLine();
             playerStats();
         }
+
+        private void backGroundTest()
```
Comment (Stackingit) [inline Program.cs:68]: You should remove this code... it was just there for testing...

did you diff before committing ? ;)

File: Program.cs [hunk 3]
```diff
@@ -356,7 +381,7 @@ private void fightDescriptionWin( Monster monster )
         {
             if (knightOrMage == "m")
             {
-                var lines = LoadCSVFile("./Config/WinDescriptions-Knight.csv");
+                var lines = LoadCSVFile("./Config/WinDescriptions-Mage.csv");
```
Comment (Stackingit) [inline Program.cs:384]: nicely spotted - I must have missed this in the last review ;)

File: Program.cs [hunk 4]
```diff
@@ -866,7 +895,13 @@ internal bool End()
...
     {
         static void Main(string[] args)
-        {      
+        {           
+            /*Console.Beep(250, 2000);
+            Console.Beep(450, 900);
+            Console.Beep(700, 3000);
+            Console.Beep(250, 2000);
+            return;
+*/
```
Comment (Stackingit) [inline Program.cs:904]: remove commented out code.

File: Program.cs [hunk 5]
```diff
@@ -620,6 +644,11 @@ public void itemShop()
             
         }
 
+        private void morseCodeMessage()
+        {
+            Console.Beep(100, 100);
+        }
+
```
Comment (Stackingit) [inline Program.cs:651]: This method is never used

**Comment by Stackingit** [comment]
There should also be a way in the commit message to link this to the open task you have in the project - lets find out how.

File: Program.cs (hunk 3 above)
Comment (SethBailey) [reply Program.cs:384]: I found that the hard way

File: Program.cs (hunk 5 above)
Comment (SethBailey) [reply Program.cs:651]: I start using this method for something else

File: Program.cs [hunk 6]
```diff
@@ -614,39 +593,134 @@ public void itemShop()
...
-                case "m" : BuyMedicine(); break;
-                case "l" : TypeWriter.WriteLine($"Bye {cottonUserName} :)",TypeWriter.Speed.Talk); break;
-                default: TypeWriter.WriteLine("Sorry but I don't understand", TypeWriter.Speed.Talk); itemShop(); break;
+                case "w": BuyWeapon(); break;
+                case "a": BuyArmour(); break;
+                case "m": BuyMedicine(); break;
+                case "l": TypeWriter.WriteLine($"Bye {cottonUserName} :)", TypeWriter.Speed.Talk); break;
+                case "h": TypeWriter.WriteLine("Sure thing");
+                morseCodeMessage();
+                shopCommanPlace();
```
Comment (Stackingit) [inline Program.cs:651]: Should be indented

File: Program.cs [hunk 7]
```diff
@@ -614,39 +593,134 @@ public void itemShop()
...
+                break;
+
+                default: TypeWriter.WriteLine("Sorry but I don't understand", TypeWriter.Speed.Talk);
+                itemShop();
```
Comment (Stackingit) [inline Program.cs:655]: indent

File: Program.cs [hunk 8]
```diff
@@ -614,39 +593,134 @@ public void itemShop()
...
+
+                switch (somthingCoolAwnser)
+                {
+                    case "yes":
+                        playerHasSeenMorse = true;
+                        TypeWriter.WriteLine(new Text("Some guy came the other day and gave me this tape (make shure you can hear it :)", Colours.Speech, TypeWriter.Speed.Talk));
+                        morseCodeMessage();
+                        Thread.Sleep(1000);
+                        TypeWriter.WriteLine("strange huh?... but anyway are you looking for");  
+                        shopCommanPlace();
```
Comment (Stackingit) [inline Program.cs:675]: You could simply put this call to shopCommonPlace() after you call somethingCool - that way your somethingCool methods knows nothing about what happens next .. so it is only doing 1 thing.

File: Program.cs [hunk 9]
```diff
@@ -614,39 +593,134 @@ public void itemShop()
...
-                                 new Text("(w)weapons, ",Colours.Attack, TypeWriter.Speed.Talk),
-                                 new Text("(a)armour ",Colours.Protection, TypeWriter.Speed.Talk),
-                                 new Text("or ",Colours.Speech, TypeWriter.Speed.Talk),
-                                 new Text("(m)medicine ",Colours.Health, TypeWriter.Speed.Talk),
-                                 new Text(":)",Colours.Speech,TypeWriter.Speed.Talk));
-            TypeWriter.WriteLine("Or (l) to leave",TypeWriter.Speed.Talk);
-            
+            if (hasPlayerBeenInShop == true)
+            {
+                somethingCool();
```
Comment (Stackingit) [inline Program.cs:606]: calling shopCommonPlace() would be better here

File: Program.cs [hunk 10]
```diff
@@ -614,39 +593,134 @@ public void itemShop()
...
+            }
+            else
+            {   
+                TypeWriter.WriteLine($"Hello {cottonUserName} ");
+                shopCommanPlace();
```
Comment (Stackingit) [inline Program.cs:611]: Spelling : Comman => Common :)

**Pull request merged**